common/changes/@microsoft/rush/master_2019-11-20-02-32.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Removing registry check from --pack, since --pack might need to run view against a registry to check if a package exists",
```

Comment:
```suggestion
      "comment": "Remove an error thrown when the --registry and --pack arguments are used on rush publish, because --pack might be required to check if a package has been published against a custom registry.",
```

Reply:
Amended with "Remove an error thrown when the --registry and --pack arguments are used on rush publish, because **--registry** might be required to check if a package has **already** been published against a custom registry." -- Edits in bold.